notary-server/notary-server.manifest.template

```diff
@@ -0,0 +1,54 @@
+[libos]
+entrypoint = "{{ self_exe }}"
+
+
+[loader]
+entrypoint = "file:{{ gramine.libos }}"
+env.MALLOC_ARENA_MAX = "1"
+env.RUST_BACKTRACE = "full"
+env.LD_LIBRARY_PATH = "/lib:{{ arch_libdir }}"
+log_level = "error"
+uid = 65534
+gid = 65534
+
+[fs]
+mounts = [
+    { path = "/lib", uri = "file:{{ gramine.runtimedir() }}" },
+    { path = "{{ arch_libdir }}", uri = "file:{{ arch_libdir }}" },
+    { path = "/fixture", uri = "file:fixture" },
+]
+
+[sgx]
+edmm_enable = true
+
+allowed_files = [
+    "file:fixture/tls",
+]
+trusted_files = [
+    "file:{{ gramine.runtimedir() }}/",
+    "file:{{ arch_libdir }}/",
+    "file:{{ self_exe }}",
+    { uri = "file:config/config.yaml" },
+    { uri = "file:fixture/notary/notary.key" },
```

Comment:
whats the reason that these fixture files need to included again? i imagine these wouldn't be used in production setting

Reply:
you bring up a good point -- really what we need is to attest our hosted notary-server; ill dm you on this